### ISSUE TITLE
Leaving out `use builtin::*` causes panic

### ISSUE THREAD

**Issue by jaylorch** [verus-lang/verus#575]
The following program:
```
//use builtin::*; // forgot to include this
use vstd::prelude::*;

verus! {

    pub struct MyType {
        x: nat,
    }

    fn main() {
    }

}
```

causes the verifier to panic. Here's a full backtrace:
```
thread 'rustc' panicked at 'no entry found for key', vir/src/recursive_types.rs:181:31
stack backtrace:
   0:     0x7fee2e95d59a - std::backtrace_rs::backtrace::libunwind::trace::h595f06c70adcc478
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/std/src/../../backtrace/src/backtrace/libunwind.rs:93:5
   1:     0x7fee2e95d59a - std::backtrace_rs::backtrace::trace_unsynchronized::h177a0149c76cdde9
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/std/src/../../backtrace/src/backtrace/mod.rs:66:5
   2:     0x7fee2e95d59a - std::sys_common::backtrace::_print_fmt::hc0701fd2c3530c58
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/std/src/sys_common/backtrace.rs:65:5
   3:     0x7fee2e95d59a - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::hd4cd115d8750fd6c
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/std/src/sys_common/backtrace.rs:44:22
   4:     0x7fee2e9bf39e - core::fmt::write::h93e2f5923c7eca08
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/core/src/fmt/mod.rs:1213:17
   5:     0x7fee2e94dbe5 - std::io::Write::write_fmt::h8162dbb45f0b9e62
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/std/src/io/mod.rs:1682:15
   6:     0x7fee2e95d365 - std::sys_common::backtrace::_print::h1835ef8a8f9066da
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/std/src/sys_common/backtrace.rs:47:5
   7:     0x7fee2e95d365 - std::sys_common::backtrace::print::hcb5e6388b9235f41
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/std/src/sys_common/backtrace.rs:34:9
   8:     0x7fee2e96012f - std::panicking::default_hook::{{closure}}::h9c084969ccf9a722
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/std/src/panicking.rs:267:22
   9:     0x7fee2e95fe6b - std::panicking::default_hook::h68fa2ba3c3c6c12f
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/std/src/panicking.rs:286:9
  10:     0x7fee2e960949 - std::panicking::rust_panic_with_hook::h8d5c434518ef298c
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/std/src/panicking.rs:688:13
  11:     0x7fee2e9606e9 - std::panicking::begin_panic_handler::{{closure}}::hf33414f5dabf6faf
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/std/src/panicking.rs:579:13
  12:     0x7fee2e95da4c - std::sys_common::backtrace::__rust_end_short_backtrace::hc50389427413bb75
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/std/src/sys_common/backtrace.rs:137:18
  13:     0x7fee2e9603f2 - rust_begin_unwind
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/std/src/panicking.rs:575:5
  14:     0x7fee2e9bbd43 - core::panicking::panic_fmt::h2de7a7938f816de8
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/core/src/panicking.rs:64:14
  15:     0x7fee2e9bbeb1 - core::panicking::panic_display::h20dbb78c57df5693
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/core/src/panicking.rs:147:5
  16:     0x7fee2e9bbe5b - core::panicking::panic_str::hb46d5710e4ab412a
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/core/src/panicking.rs:131:5
  17:     0x7fee2e9bbac6 - core::option::expect_failed::h0b5d04c30548770b
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/core/src/option.rs:1924:5
  18:     0x556ae36b0f98 - <std::collections::hash::map::HashMap<K,V,S> as core::ops::index::Index<&Q>>::index::h584efa0711e5b206
  19:     0x556ae36c1cf6 - vir::recursive_types::check_positive_uses::h326d2b5f037ef4fa
  20:     0x556ae36c2607 - vir::recursive_types::check_recursive_types::hbb2274b451be4a00
  21:     0x556ae37ebd0d - vir::well_formed::check_crate::hf901797fb7fed408
  22:     0x556ae33f6f9f - rust_verify::verifier::Verifier::construct_vir_crate::hd1d1a7a80cf857d8
  23:     0x556ae34a49cc - verus_rustc_interface::passes::QueryContext::enter::hf060498564a015b6
  24:     0x556ae33f7f25 - <rust_verify::verifier::VerifierCallbacksEraseMacro as verus_rustc_driver::Callbacks>::after_expansion::h4580e795c27adae7
  25:     0x556ae3591370 - verus_rustc_interface::queries::<impl verus_rustc_interface::interface::Compiler>::enter::hd8ac72a3c8c0c46a
  26:     0x556ae35af26a - rustc_span::with_source_map::h2122596747b16d25
  27:     0x556ae35561b5 - scoped_tls::ScopedKey<T>::set::h86c638eb05fc5fae
  28:     0x556ae3549470 - std::sys_common::backtrace::__rust_begin_short_backtrace::h549de802b86bff6b
  29:     0x556ae353a24d - core::ops::function::FnOnce::call_once{{vtable.shim}}::hc33b409d9eaebf7a
  30:     0x7fee2e96a823 - <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once::h1c0f3664d7ced314
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/alloc/src/boxed.rs:1988:9
  31:     0x7fee2e96a823 - <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once::h67647c21c6c4968a
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/alloc/src/boxed.rs:1988:9
  32:     0x7fee2e96a823 - std::sys::unix::thread::Thread::new::thread_start::h355d348ba593a22c
                               at /rustc/2c8cc343237b8f7d5a3c3703e3a87f2eb2c54a74/library/std/src/sys/unix/thread.rs:108:17
  33:     0x7fee2e814609 - start_thread
  34:     0x7fee2e5e8133 - clone
  35:                0x0 - <unknown>
```

**Comment by tjhance** [comment]
fixed in bbe9b99d40b9d639a77e0cdab9cfc7bf7beba2ce

**Comment by jaylorch** [comment]
Thanks for fixing that so quickly, @tjhance!
